doc/tutorials/content/index.rst

```diff
@@ -97,16 +97,16 @@ Basic Usage
      .. |mi_3| image:: images/pcl_ccmake.png
                :height: 100px
 
-  * :ref:`compiling_pcl_dependencies_windows`
+  * :ref:`pcl_vcpkg_windows`
 
      ======  ======
-     |mi_4|  Title: **Compiling PCL's dependencies from source on Windows**
+     |mi_4|  Title: **Install PCL using VCPKG**
```

Comment:
Might be good to mention that this is for Windows (e.g. "Install PCL on Windows using VCPKG"

Reply:
Actually, it can probably also be used on linux as well - so it might be a cross operating system tutorial. However, bash commands would probably be used more often, than gui windows - though they are also possible with VCPKG + CMake.

Reply:
Yes, the vcpkg commands are probably similar on Linux, but otherwise the tutorial is directed towards Windows (e.g. where to clone the vcpkg repo). And my thought was also that everyone who wants to know how to install PCL on Windows, finds this tutorial easily